### DISCUSSION TITLE
How to debug using sudo

### DISCUSSION THREAD

**Discussion by Giancarlo-Ma** [microsoft/vscode-cmake-tools#2445] [Q&A]
Some code are required sudo permission, so when click the debug button, It just shows that I have no permission as a regular user

**Comment by bobbrow** [comment]
We don't currently have a way to do this in the extension right now.  I will convert this into a feature request.

**Comment by AlexeyVT** [comment]
> Some code are required sudo permission, so when click the debug button, It just shows that I have no permission as a regular user

as a possible solution

$ cat /usr/bin/lldb-mi
sudo /lib/llvm-7/bin/lldb-mi "$@"

or 

cat /usr/bin/gdb
#!/bin/sh
sudo /usr/bin/gdb-orig "$@"
